.env.example

```
@@ -1,2 +1,11 @@
+PH_PROJECT_KEY=
+PH_HOST=https://us.i.posthog.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The PH_HOST key should go before the PH_PROJECT_KEY key

(UnorderedKey)